src/herbivoresAndCarnivores.js

```diff
@@ -1,16 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  };
+};
+
+Animal.alive = [];
 
 class Herbivore extends Animal {
-  // write your code here
-}
+  constructor(name, health, hidden) {
+    super(name);
+    this.hidden = false;
+
+    Animal.alive.push(this);
+  };
+
+  hide() {
+    this.hidden = !this.hidden;
+  };
+};
 
 class Carnivore extends Animal {
-  // write your code here
-}
+  constructor(name) {
+    super(name);
+
+    Animal.alive.push(this);
+  };
+
+  bite(animal) {
+    if (animal.hidden === false) {
```

Comment:
```suggestion
    if (animal.hidden === false) {
```
Тут ще також треба додати перевірку на те, чи жертва - травоядна. Хижаки можуть кусати лише травоядних, але не інших хижаків